src/registries/npm-registry.js

```diff
@@ -12,7 +12,7 @@ import {addSuffix, removePrefix} from '../util/misc';
 import isRequestToRegistry from './is-request-to-registry.js';
 
 const defaults = require('defaults');
-const userHome = require('user-home');
+const userHome = require('os').homedir();
```

Comment:
Yeah, I think we should use one homedir definition everywhere.
Use your best judgement and lowercase letters, something with words user, home and dir maybe :)